ocean_lib/assets/test/test_utils.py

```diff
@@ -115,12 +108,13 @@ def test_fail_generate_trusted_algo_dict():
 
 def test_utilitary_functions_for_trusted_algorithm_publishers(publisher_ocean_instance):
     """Tests adding/removing trusted algorithms in the DDO metadata."""
-    publisher = get_publisher_wallet()
-    ddo = get_registered_ddo_with_compute_service(
-        publisher_ocean_instance, publisher, trusted_algorithm_publishers=["0xabc"]
-    )
-    wait_for_ddo(publisher_ocean_instance, ddo.did)
-    assert ddo is not None, "DDO is not found in cache."
+    ddo = get_sample_ddo_with_compute_service()
+    compute_service = ddo.get_service(ServiceTypes.CLOUD_COMPUTE)
+    privacy_values = compute_service.attributes["main"].get("privacy")
+    if not privacy_values:
```

Comment:
This `if` statement should be unnecessary.  Please remove it and update `ddo_with_compute_service.json` to contain a well-formed compute service with the `publisherTrustedAlgorithmPublishers` key defined.